src/Trakx.Kaiko.ApiClient.Tests/Integration/MarketDataTests.cs

```diff
@@ -5,64 +5,76 @@ namespace Trakx.Kaiko.ApiClient.Tests;
 
 public class MarketDataTests : IntegrationTestsBase
 {
+    private const int ExpectedResults = 3;
+    private readonly DateTime _endTime;
+    private readonly DateTime _startTime;
+
     public MarketDataTests(KaikoApiFixture fixture, ITestOutputHelper output)
         : base(fixture, output)
     {
+        _endTime = DateTime.UtcNow.Date;
+        _startTime = _endTime.AddDays(-ExpectedResults);
     }
 
     [InlineData(EnabledServices.AggregateOhlcv)]
     [Theory]
     public async Task AggregatesClient_GetAggregateOhlcvAsync_should_return_data(bool serviceEnabled)
     {
```

Comment:
If I try to write a Fact and then check the `EnabledServices.AggregateOhlcv`, the IDE will complain that the statement will always be true or false. By hiding behind `InlineData`, I trick the IDE 😅 
Is there a better way to do a selective "skip test" using a flag?

Reply:
I think it's not worth it, but it seems we could create a custom FactAttribute.
There is an example in this link to get "inspired" hehe: https://josephwoodward.co.uk/2019/01/skipping-xunit-tests-based-on-runtime-conditions
For example, it could be like `ConditionalSkip(bool enabled)`.
If this is interesting, maybe could be available in `Trakx.Common.Testing` lib. 🤔